### PR TITLE
Repose ninja sprint97

### PR DESCRIPTION
**Pull request by wdschei**
Bio:
Bill is a Software Developer at Rackspace that is currently working on the open source Repose project
and has almost two decades of professional experience in Software and Systems design.
Prior to joining Rackspace and after eight years of active duty service in the USAF,
he worked primarily on Iridium Based Friendly Force Tracking Solutions for the US Military.

Social Media:
GitHub: https://github.com/wdschei
LinkedIn: https://www.linkedin.com/in/WilliamScheidegger
G+: https://www.google.com/+WilliamScheidegger

Editing Preference:
Feel free to edit as needed.

Posting Preference:
Desired date to publish: 2015-07-22

### CODE REVIEW

File: _posts/2015-07-22-repose-ninja-097.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    - OpenStack
+    - OpenStack-Keystone
+    - API Management
+---
+
+![Alt text here]({% asset_path repose_logo.png %})
+
+This is the first in a series of Sprint-ly (Bi-Weekly) posts written by the [Repose][repose] Ninja on Duty.
+
+What is [Repose][repose] and why should you care about that happy little [Narwahl][narwahl]?
```
Comment (catlook) [inline _posts/2015-07-22-repose-ninja-097.md]: hat is [Repose][repose], and why should you ...

File: _posts/2015-07-22-repose-ninja-097.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+* ... developed (copy/pasted) the same functionality again and again as you created each new service?
+
+Well, [Repose][repose] can help with all of these things and many more.
+
+<!-- more -->
+
+# What is Repose and how do I get help
+Since this is the first of the [Repose][repose] Ninja on Duty Sprint-ly posts, I thought I should introduce
+[Repose][repose] a bit.
+[Repose][repose] is a Reverse Proxy that helps protect/manage backend Origin Services
```
Comment (catlook) [inline _posts/2015-07-22-repose-ninja-097.md]: ... backend Origin Services,

File: _posts/2015-07-22-repose-ninja-097.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+and the [Repose][repose] Ninja on Duty is a position that all of the [Repose][repose] developers rotate through.
+While assigned as the Ninja on Duty, we monitor
```
Comment (catlook) [inline _posts/2015-07-22-repose-ninja-097.md]: ... we monitor:

File: _posts/2015-07-22-repose-ninja-097.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+* our public [Freenode IRC channel #repose][irc]
+* email to the [ReposeCore@Rackspace.com][email] list
+* questions/comments on our [Wiki][wiki]
+
+So if you ever have any questions about how to set up, configure, and/or upgrade [Repose][repose],
```
Comment (catlook) [inline _posts/2015-07-22-repose-ninja-097.md]: So, if you ever ...

File: _posts/2015-07-22-repose-ninja-097.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+then feel free to reach out to us at any of these methods.
```
Comment (catlook) [inline _posts/2015-07-22-repose-ninja-097.md]: ... to us by using any of ..

File: _posts/2015-07-22-repose-ninja-097.md [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+Oh yeah, I almost forgot, the [Repose][repose] Ninja on Duty is also responsible for writing one of these posts.
+
+Now that you know a little bit about who we are, you are probably wondering what [Repose][repose] actually does or more
```
Comment (catlook) [inline _posts/2015-07-22-repose-ninja-097.md]: ... actually does or, more

File: _posts/2015-07-22-repose-ninja-097.md [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+importantly how it can help you.
```
Comment (catlook) [inline _posts/2015-07-22-repose-ninja-097.md]: importantly, how it ...

File: _posts/2015-07-22-repose-ninja-097.md [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+Most installations are seeing around 4-6 ms added to response times, with some minimal installations down to only 1 ms.
+There are filters for:
+
+* Rate Limiting
+* API Management and Validation
+* Header Management and Translation/Extraction
+* Cloud Auditing Data Federation (CADF)/User Access Events (UAE) via the HERP and DERP filter combination
+* and many more.
+
+Some of the [Repose][repose] filters are tailored or specifically designed for Rackspace API's,
```
Comment (catlook) [inline _posts/2015-07-22-repose-ninja-097.md]: ... tailored, or specifically designed, for Rackspace ...

File: _posts/2015-07-22-repose-ninja-097.md [hunk 9]
```diff
@@ -0,0 +1,101 @@
...
+What can you do?
+
+# Hello World project to the rescue
+The [Repose Hello World project](https://github.com/rackerlabs/repose-hello-world) walks you through how you can create a
+custom filter bundle to add new functionality to [Repose][repose].
+This is a good option if you need something quick to prototype a new functionality before the [Repose][repose] team has
+time to add it or if you need truly custom behavior that will never be added to the baseline.
+
+Currently this project has examples in Java, Scala, and Groovy.
+Using these examples any language that executes directly on the Java Virtual Machine (JVM) could be added.
```
Comment (catlook) [inline _posts/2015-07-22-repose-ninja-097.md]: Using these examples, any language  ...

File: _posts/2015-07-22-repose-ninja-097.md [hunk 10]
```diff
@@ -0,0 +1,101 @@
...
+There are even plans to expand the Hello World project to include several scripting languages that have been
```
Comment (catlook) [inline _posts/2015-07-22-repose-ninja-097.md]: ... to include the following scripting languages  ...

File: _posts/2015-07-22-repose-ninja-097.md [hunk 11]
```diff
@@ -0,0 +1,101 @@
...
+ported or bridged to the JVM:
+
+* Python (Jython)
+* Ruby (JRuby)
+* Lua (LuaJ)
+
+Let me close with, [Repose][repose] is Open Source and we are more than happy to accept external [Pull Requests][github].
```
Comment (catlook) [inline _posts/2015-07-22-repose-ninja-097.md]: Let me close with this: [Repose][repose] is Open Source ...

File: _posts/2015-07-22-repose-ninja-097.md [hunk 12]
```diff
@@ -0,0 +1,101 @@
...
+Also, if you come across something that you would like us to go into in a future [Repose][repose] Ninja on Duty post,
+then please contact us on [IRC][irc], [email][email], or the [Wiki][wiki].
+
+[repose]: http://www.OpenRepose.org/
+[wiki]: http://wiki.OpenRepose.org/
+[github]: https://github.com/rackerlabs/repose
+[email]: mailto:ReposeCore@Rackspace.com
+[irc]: irc://irc.freenode.net:6667/repose
+[narwahl]: https://www.worldwildlife.org/stories/unicorn-of-the-sea-narwhal-facts
```
Comment (catlook) [inline _posts/2015-07-22-repose-ninja-097.md]: [Ed note:  Nice article!!  Feel free to use any of my edits that you like and discard the rest!!  -- Cat]

**Comment by wdschei** [comment]
Thanks, Cat. Just pushed the suggested edits.

**Pull request merged**